pages/Product/streams.js

```diff
@@ -0,0 +1,20 @@
+import { routeDidEnter } from '@shopgate/pwa-common/streams/history';
+import { routeWillEnter$ } from '@shopgate/pwa-common/streams/router';
+import { ITEM_PATH } from '@shopgate/pwa-common-commerce/product/constants';
+
+export const productWillEnter$ = routeWillEnter$
+  .filter(({ action }) => (
+    action.route.pathname.startsWith(ITEM_PATH) &&
```

Comment:
Please change the ITEM_PATH constant to include the :productId param. Then you can just test the pattern against it.

Reply:
Done